examples/exporting_data.ipynb

```diff
@@ -449,6 +449,39 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import networkx as nx\n",
+    "graph = traj_contacts.residue_contacts.to_networkx()\n",
+    "nx.draw(graph)"
```

Comment:
Just a note, this line errors out for` networkx<=2.3` and matplotlib>=`3.3.0`. We might need want force some recent version 
```python
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-6-78807c782959> in <module>
      1 import networkx as nx
      2 graph = traj_contacts.residue_contacts.to_networkx()
----> 3 nx.draw(graph)

~/miniconda3/envs/contact_map/lib/python3.8/site-packages/networkx/drawing/nx_pylab.py in draw(G, pos, ax, **kwds)
    126 
    127     try:
--> 128         draw_networkx(G, pos=pos, ax=ax, **kwds)
    129         ax.set_axis_off()
    130         plt.draw_if_interactive()

~/miniconda3/envs/contact_map/lib/python3.8/site-packages/networkx/drawing/nx_pylab.py in draw_networkx(G, pos, arrows, with_labels, **kwds)
    278 
    279     node_collection = draw_networkx_nodes(G, pos, **kwds)
--> 280     edge_collection = draw_networkx_edges(G, pos, arrows=arrows, **kwds)
    281     if with_labels:
    282         draw_networkx_labels(G, pos, **kwds)

~/miniconda3/envs/contact_map/lib/python3.8/site-packages/networkx/drawing/nx_pylab.py in draw_networkx_edges(G, pos, edgelist, width, edge_color, style, alpha, arrowstyle, arrowsize, edge_cmap, edge_vmin, edge_vmax, ax, arrows, label, node_size, nodelist, node_shape, connectionstyle, **kwds)
    577     edge_pos = np.asarray([(pos[e[0]], pos[e[1]]) for e in edgelist])
    578 
--> 579     if not cb.iterable(width):
    580         lw = (width,)
    581     else:

AttributeError: module 'matplotlib.cbook' has no attribute 'iterable'

```

Reply:
I feel like this is an issue with NetworkX compatibility, and is hard for us to solve. Since neither NetworkX nor matplotlib are requirements (you just get extra functionality if you have them installed), we can't pin version numbers. Plus, the issue is that internally the NetworkX imports aren't compatible with certain versions of matplotlib, which means the problem is with the NetworkX install. However, **we** are compatible with `networkx<=2.3`, as well as with `matplotlib>=3.3.0`. It's just that NetworkX allows itself to be installed with an incompatible matplotlib. Maintaining a list of incompatible version pairs is their job, not ours.

(Aside: IIRC, NetworkX also has similar issues with Python 3.9+ due to older versions of NetworkX using `from fractions import gcd`, which was removed in 3.9. Similar problem; similar answer: Not our job to check that NetworkX version is compatible with Python version.)

Reply:
That is a fair measure (as long as we are compatible don't force any versions), just wanted to make sure you were aware of possible issues if users come back